### DISCUSSION TITLE
test failing due to improper indentation

### DISCUSSION THREAD

**Discussion by jackiebgregory** [nss-evening-cohort-14/evening-client-side#34] [Help Me!] [answered]
**Your Name:** Jackie

**Have you searched the other help tickets?** Yes.

**Describe your issue:** On the js-practice-1 assignment, I got through 1-6 just fine. I'm completely stuck on 7 (and the rest). I think once I understand 7, I will be able to get the rest. During class, I understood and was following along and keeping up, so I'm not sure why I am so confused now.  

**List any error messages that appear:** When I run ```npm test main.test.js```, 1-6 are correct, and the 7 is not.

**What have you googled? (Paste links to the things you have found from your google search - You must include at least 3 links)**
https://www.w3schools.com/jsref/jsref_return.asp
https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Return_values
https://stackoverflow.com/questions/14076448/returning-a-string-in-javascript
I think this one is the closest to getting me on the right track, but I'm still not seeing it. https://www.codecademy.com/forum_questions/505b318b78e591000200e39d
I've googled more than 3 links, but most have used examples much more complicated than the ones we are working on here, and so I can't figure out how to apply them.

**What have you tried? Tell us everything you have tried.**
-Reread the directions. 
```
In the next 22 problems you will compete the function. All of your code will go inside of the function braces.
//Make sure you use return when the prompt asks you to.
//hint: console.log() will NOT work.
//Do not change any of the function names
```
-Reread my notes.
-Reread js-intro that we did in class. 
-Reread Chapter 5-Javascript in Book 1 Foundations.
-Returned to the Javascript section of freecodecamp.
-Tested the following ideas:

```function returnString(str) {
   //simply return the string provided: str

}

function returnString(str) {
   //simply return the string provided: str
return str;
}

function returnString(str) {
   //simply return the string provided: str
returnString();
}

function returnString('str') {
   //simply return the string provided: str
return returnString();
}

function returnString(str) {
   //simply return the string provided: str
returnString('Hello!");
}

function returnString(str) {
   //simply return the string provided: str
const str = "Hello!";
return str;
}

function returnString(str) {
   //simply return the string provided: str
return str("Hello!");
}
```

I feel like this is a question I shouldn't even need to ask, but I can't figure out how to move forward with the assignment if I don't!

**Comment by chiestroud** [comment, marked as answer]
Hello Jackie, I think you have the correct answer in the first try, but each nested statement in the function should be indented :)

**Reply by TrinityChristiana** [reply]
Hi Chie! Thank you for helping a teammate. Remember to put the time you spent helping them in CodeTracker.

**Reply by chiestroud** [reply]
Thank you Trinity!

**Comment by jackiebgregory** [comment]
Thank you!! That was it! I was both hoping and afraid it was that simple of an issue.... Thank you!!

**Reply by chiestroud** [reply]
Yay!
